sdk/storage/azure-storage-queue/azure/storage/queue/_shared_access_signature.py

```diff
@@ -21,6 +21,7 @@
         AccountSasPermissions,
         QueueSasPermissions
     )
+    from typing import Any
```

Comment:
We shouldn't import anything from `typing` inside a `TYPE_CHECKING` clause, that's really just for either expensive imports or circular imports.